spec/controllers/omniauth_callbacks_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Users::OmniauthCallbacksController do
```

Comment:
Could we also check here that `Users::Omniauth.new(auth).user` was called? We can go with [`expect.to receive`](https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages) syntax. It might be easy.

Reply:
I was unsuccessful in doing this. I have checked the call of the method in debugging mode. It has been called out of this controller. But this syntax doesn't work for some reason. I've spent a couple of hours trying to find the solution, but at the end, I have decided it not worth of trying anymore. 

If you could show me the solution here, I would be happy. Thnx

Reply:
@bazarnyi my bad, I thought it would be easy. Try this snippet. If it doesn't work, leave test as is.
```ruby
before do
  stub_omniauth_env 'github'
  expect_any_instance_of(Users::Omniauth).to receive(:user)
  do_request
end
```